audio_separator/utils/cli.py

```diff
@@ -145,11 +145,11 @@ def main():
         logger.info(f"Model {args.model_filename} downloaded successfully.")
         sys.exit(0)
 
-    if not hasattr(args, "audio_file"):
+    if not hasattr(args, "audio_files"):
         parser.print_help()
         sys.exit(1)
 
-    logger.info(f"Separator version {package_version} beginning with input file: {args.audio_file}")
+    logger.info(f"Separator version {package_version} beginning with input file(s): {", ".join(args.audio_files)}")
```

Comment:
FYI there's a syntax error in this f-string which breaks the tests;

`poetry run pytest`

```
======================================================================================================= ERRORS =======================================================================================================
______________________________________________________________________________________ ERROR collecting tests/unit/test_cli.py _______________________________________________________________________________________
../../miniforge3/envs/audio-separator-clean-py311/lib/python3.11/site-packages/_pytest/python.py:493: in importtestmodule
    mod = import_path(
../../miniforge3/envs/audio-separator-clean-py311/lib/python3.11/site-packages/_pytest/pathlib.py:582: in import_path
    importlib.import_module(module_name)
../../miniforge3/envs/audio-separator-clean-py311/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1206: in _gcd_import
    ???
<frozen importlib._bootstrap>:1178: in _find_and_load
    ???
<frozen importlib._bootstrap>:1149: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:690: in _load_unlocked
    ???
../../miniforge3/envs/audio-separator-clean-py311/lib/python3.11/site-packages/_pytest/assertion/rewrite.py:184: in exec_module
    exec(co, module.__dict__)
tests/unit/test_cli.py:3: in <module>
    from audio_separator.utils.cli import main
E     File "/Users/andrew/Projects/python-audio-separator/audio_separator/utils/cli.py", line 152
E       logger.info(f"Separator version {package_version} beginning with input file(s): {", ".join(args.audio_files)}")
E                                                                                         ^
E   SyntaxError: f-string: expecting '}'
============================================================================================== short test summary info ===============================================================================================
ERROR tests/unit/test_cli.py
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
================================================================================================== 1 error in 1.32s ==================
```